types/node/index.d.ts

```diff
@@ -5894,10 +5894,10 @@ declare module "crypto" {
     export function pseudoRandomBytes(size: number): Buffer;
     export function pseudoRandomBytes(size: number, callback: (err: Error | null, buf: Buffer) => void): void;
 
-    export function randomFillSync(buffer: ArrayBufferView, offset?: number, size?: number): ArrayBufferView;
-    export function randomFill(buffer: ArrayBufferView, callback: (err: Error | null, buf: ArrayBufferView) => void): void;
-    export function randomFill(buffer: ArrayBufferView, offset: number, callback: (err: Error | null, buf: ArrayBufferView) => void): void;
-    export function randomFill(buffer: ArrayBufferView, offset: number, size: number, callback: (err: Error | null, buf: ArrayBufferView) => void): void;
+    export function randomFillSync<T extends Buffer|ArrayBufferView>(buffer: T, offset?: number, size?: number): T;
```

Comment:
style note: can you add spaces around `|` for consistency with other code.

Reply:
I don't think that `Buffer|` is needed at all as `ArrayBufferView` includes `Buffer` - if not the definition of quite some APIs in crypto would be wrong.